bin/nib-conform

```diff
@@ -0,0 +1,17 @@
+#!python
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""
+Conform a volume to a new shape and/or voxel size.
+"""
+
+from nibabel.cmdline.conform import main
+
+if __name__ == '__main__':
+    main()
```

Comment:
I don't think we install these files anywhere...

Reply:
should I remove this file?

Reply:
I think so. I'd try removing and make sure `pip install .` produces a working `nib-conform` command.